Documentation/ApiOverview/Localization/LocalizationApi/_LanguageService.rst.txt

```diff
@@ -41,15 +45,22 @@
       Main and most often used method.
       
       Resolve strings like these:
-           'LLL:EXT:core/Resources/Private/Language/locallang_core.xlf:labels.depth_0'
-      This looks up the given .xlf file path in the 'core' extension for label 'labels.depth_0'
+      
+      ..  code-block:: php
+          
+               'LLL:EXT:core/Resources/Private/Language/locallang_core.xlf:labels.depth_0'
+          
+      
+      This looks up the given .xlf file path in the 'core' extension for label labels.depth_0
       
       :param string $input: Label key/reference
       :returntype: string
       
    .. php:method:: includeLLFile(string $fileRef)
    
       Includes locallang file (and possibly additional localized version if configured for)
-      Read language labels will be merged with $LOCAL_LANG (if $setGlobal = TRUE).
+      Read language labels will be merged with $LOCAL_LANG (if $setGlobal === true).
```

Comment:
Comparing with true does not really make sense, does it? Maybe turn it into a sentence?